paasta_tools/vitesscluster_tools.py

```diff
@@ -756,6 +771,27 @@ def get_keyspaces(self) -> List[KeyspaceConfigDict]:
     def get_update_strategy(self) -> Dict[str, str]:
         return {"type": "Immediate"}
 
+    def get_prometheus_pushgateway_config(self) -> PrometheusPushGatewayConfigDict:
+        config = PrometheusPushGatewayConfigDict(
+            name="prometheusPushGateway",
+            image="prometheus/pushgateway",
```

Comment:
just in case we do end up going with this: this would need to be an internally mirrored image on docker-paasta

Reply:
i also don't think we'd want to run our own pushgateway here - we'd want to use the one that PE manages 

(although, you'd want to speak to them before using since pushgateway is not generally something that we encourage/support very much)

Reply:
Ah do you mean this docker-images/pushgateway? https://sourcegraph.yelpcorp.com/docker-images/pushgateway

Reply:
if you meant the internally mirrored image: yea, we'd need to check if that's been pushed to docker-paasta

if you meant the PE-managed pushgateway: y/pushgateway :)